src/daemon/main.c

```diff
@@ -1411,7 +1411,12 @@ int unittest_prepare_rrd(char **user) {
     return 0;
 }
 
-int main(int argc, char **argv) {
+#ifdef OS_WINDOWS
+int netdata_main(int argc, char **argv)
```

Comment:
A comment: I understand we are still not using a [WinMain](https://learn.microsoft.com/en-us/windows/win32/learnwin32/winmain--the-application-entry-point) as an entry point, but we will have it one day.